### ISSUE TITLE
Can't convert StringValues to List<string>

### ISSUE THREAD

**Issue by maxlm** [agileobjects/AgileMapper#50]
I have following class  as target: 
```
    class MyDTO
    {
        public string widgetId { get;set; }

        public string clientId { get; set; }

        public string[] testPayload { get; set; }
    }
```
And 
`Dictionary<string, StringValues>` as source.

Mapping from `StringValues` to `string` works perfect. 

But when I'm trying to map `StringValues` to `string[]`, I'm getting following: 
```
System.ArgumentException: Expression of type 'Microsoft.Extensions.Primitives.StringValues' cannot be used for label of type 'System.Collections.Generic.List`1[System.Object]'
   at System.Linq.Expressions.Expression.ValidateGotoType(Type expectedType, Expression& value, String paramName)
   at System.Linq.Expressions.Expression.ValidateGoto(LabelTarget target, Expression& value, String targetParameter, String valueParameter, Type type)
   at System.Linq.Expressions.Expression.Label(LabelTarget target, Expression defaultValue)
   at AgileObjects.AgileMapper.ObjectPopulation.Enumerables.Dictionaries.SourceInstanceDictionaryAdapter.GetSourceValues()
   at AgileObjects.AgileMapper.ObjectPopulation.Enumerables.EnumerablePopulationBuilder.AssignSourceVariableFromSourceObject()
   at AgileObjects.AgileMapper.ObjectPopulation.Enumerables.CopySourceEnumerablePopulationStrategy.GetEnumerablePopulation(EnumerablePopulationBuilder builder, IObjectMappingData mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.Enumerables.EnumerableMappingExpressionFactory.<GetObjectPopulation>d__5.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at AgileObjects.AgileMapper.ObjectPopulation.MappingExpressionFactoryBase.Create(IObjectMappingData mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMapperFactory.Create[TSource,TTarget](ObjectMappingData`2 mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMappingData`2.get_Mapper()
   at AgileObjects.AgileMapper.ObjectPopulation.MappingFactory.GetInlineMappingBlock(IObjectMappingData mappingData, MappingValues mappingValues, Expression createMappingDataCall)
   at AgileObjects.AgileMapper.DataSources.DataSourceFinder.GetFinalDataSource(IDataSource foundDataSource, Int32 dataSourceIndex, IChildMemberMappingData childMappingData)
   at AgileObjects.AgileMapper.DataSources.DataSourceFinder.<EnumerateDataSources>d__5.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at AgileObjects.AgileMapper.DataSources.DataSourceFinder.FindFor(IChildMemberMappingData childMappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.MemberPopulationFactory.Create(QualifiedMember targetMember, IObjectMappingData mappingData)
   at System.Linq.Utilities.<>c__DisplayClass2_0`3.<CombineSelectors>b__0(TSource x)
   at System.Linq.Enumerable.SelectArrayIterator`2.MoveNext()
   at AgileObjects.AgileMapper.ObjectPopulation.ComplexTypes.PopulationExpressionFactoryBase.<GetPopulationsAndCallbacks>d__2.MoveNext()
   at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at AgileObjects.AgileMapper.ObjectPopulation.ComplexTypes.PopulationExpressionFactoryBase.<GetPopulation>d__0.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at AgileObjects.AgileMapper.ObjectPopulation.MappingExpressionFactoryBase.Create(IObjectMappingData mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMapperFactory.Create[TSource,TTarget](ObjectMappingData`2 mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMappingData`2.get_Mapper()
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMapperFactory.<>c__7`2.<GetOrCreateRoot>b__7_0(ObjectMapperKeyBase key)
   at AgileObjects.AgileMapper.Caching.ArrayCache`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMapperFactory.GetOrCreateRoot[TSource,TTarget](ObjectMappingData`2 mappingData)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMappingData`2..ctor(TSource source, TTarget target, Nullable`1 enumerableIndex, ObjectMapperKeyBase mapperKey, IMappingContext mappingContext, IObjectMappingData declaredTypeMappingData, IObjectMappingData parent)
   at AgileObjects.AgileMapper.ObjectPopulation.ObjectMappingDataFactory.ForRootFixedTypes[TSource,TTarget](TSource source, TTarget target, IMappingContext mappingContext)
   at AgileObjects.AgileMapper.MappingExecutor`1.PerformMapping[TTarget](TTarget target)
   at AgileObjects.AgileMapper.MappingExecutor`1.ToANew[TResult](Expression`1[] configurations)
```

**Comment by SteveWilkes** [comment]
Thanks very much for the feedback - I've found the issue and will have a fix for it in the next release later this week.

**Comment by maxlm** [comment]
yeah, that worked. Thank you. 
AgileMapper is awesome! Keep doing great job!

**Comment by SteveWilkes** [comment]
Glad to hear it, and thanks very much! The fix will be in a new version probably today, maybe tomorrow.

**Comment by SteveWilkes** [comment]
This is fixed in [v0.20](https://www.nuget.org/packages/AgileObjects.AgileMapper).

Thanks again for the feedback :)
